sass/components/_ciq-chart-setting.scss

```diff
@@ -13,6 +13,9 @@
         > .icon-back-menu {
             display: inline-block;
             cursor: pointer;
+            @include themify($themes) {
+                color: themed('ChartSettingHeadText');
```

Comment:
I can't find this color in themes.scss file, which line is it in?

Reply:
`ChartSettingHeadText`

Reply:
oops, it should be `DialogSettingHeadText`.